internal/tools/tool/ads_darwin.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package tool
+
+import (
+	"os"
+	"path/filepath"
+)
+
+func (t *AzureDataStudio) searchLocations() []string {
+	userProfile := os.Getenv("HOME")
+
+	return []string{
+		filepath.Join("/", "Applications", "Azure Data Studio - Insiders.app"),
+		filepath.Join(userProfile, "Downloads", "Azure Data Studio - Insiders.app"),
+		filepath.Join("/", "Applications", "Azure Data Studio.app"),
+		filepath.Join(userProfile, "Downloads", "Azure Data Studio.app"),
+	}
+}
+
+func (t *AzureDataStudio) installText() string {
+	return `Download the latest .zip from:
+
+    https://go.microsoft.com/fwlink/?linkid=2151311
+
+More information can be found here:
+
+    https://docs.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?#get-azure-data-studio-for-macos`
```

Comment:
>n-us [](http://example.com/codeflow?start=32&length=4)

remove lang id from URLs

Reply:
didn't know you could do that, done through the project